SharedSources/MediaDataSource.swift

```diff
@@ -27,7 +28,13 @@ public class MediaDataSourceAndDelegate:NSObject, UICollectionViewDataSource, UI
     }
 
     public func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
-        return Int(services.mediaDataSource.numberOfFiles())
+        let numItems = Int(services.mediaDataSource.numberOfFiles())
+        if numItems == 0 {
```

Comment:
can we write 
Bool hasMedia = numItems > 0
collectionView.backgroundView?.isHidden = hasMedia